### PR TITLE
Get initial inspector rendering

### PR DESCRIPTION
**Pull request by hatched**
The inspector is now rendered when an application is added to the canvas and you can navigate to the first level.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6142/
Test FAILed.

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1347,7 +1345,7 @@ YUI.add('juju-gui', function(Y) {
             envResolved={this.env.resolved.bind(this.env)}
             serviceRelations={
               relationUtils.getRelationDataForService(this.db, service)}
-            addGhostAndEcsUnits={utils.addGhostAndEcsUnits.bind(
+            addGhostAnlocalTypedEcsUnits={utils.addGhostAndEcsUnits.bind(
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/app.js]: Did you mean to update this? I couldn't see any changes to the component prop name.

File: jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js [hunk 2]
```diff
@@ -192,57 +187,37 @@ YUI.add('service-overview', function() {
...
-            metadata: {
+          gui: {
+            inspector: {
               id: service.get('id'),
-              activeComponent: 'config'
-            }
-          }
-        }
-      });
+              activeComponent: 'config'}}}});
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js]: `}}}});` really!? :)

File: jujugui/static/gui/src/app/state/state.js [hunk 3]
```diff
@@ -127,23 +127,31 @@ const State = class State {
...
     @type {Object}
   */
-  get appState() {
+  get current() {
     return this._appStateHistory[this._appStateHistory.length-1];
   }
 
   /**
+    The object representing the application history.
+    @type {Array}
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/state/state.js]: The comment says object the type is array?

File: jujugui/static/gui/src/app/app.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: Oops fixed

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6143/
Test FAILed.

**Pull request merged**